modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -101,15 +101,15 @@ Runtime Fabric on VMs / Bare Metal supports log forwarding to:
 ** Splunk
 ** CloudWatch
 
-[NOTE]
-====
-* Only one external log forwarding configuration is supported at any one time. However, you can enable Anypoint log forwarding and an external log forwarding service at the same time.
+=== Limitations for External Log Forwarding
```

Comment:
@hegderohit89 this is the new section

Reply:
This is great, we can extend this section in the future